docs/cudf/source/developer_guide/cudf_pandas.md

```diff
@@ -38,6 +38,17 @@ Note that users should never have to interact with the wrapped objects directly
 This code is purely for demonstrative purposes.
 ```
 
+We can check whether an object is a proxy type with `cudf.pandas.is_proxy_object`.
+```
```

Comment:
Missing a language
```suggestion
```python
```
But maybe this would be better to include directly in the snippet above instead so that it's clear we're using the same objects?

Reply:
I don't mind moving it to the same block. Although, I think there are [other places](https://docs.rapids.ai/api/cudf/stable/developer_guide/library_design/#examples) in the docs where variables are defined and used in different code snippets.

Reply:
Yeah it's definitely done elsewhere too, I just noted this because in the diff I didn't initially realize what s1/s2/s3 were since they weren't part of the diff. No real need to change this, but since you already did let's keep it merged.